test/src/hooks/use-form.test.js

```diff
@@ -352,7 +380,7 @@ describe('useForm hook', () => {
       expect(result.current.state.fields.values).toEqual({ foo: 'bar' });
     });
 
-    it('should set all fields to touched and dirty', async () => {
+    it('should set all fields to touched', async () => {
```

Comment:
I'm not sure about this change. What do you think?

Reply:
sounds good , it should be like